packages/snowpack-plugin/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import path from "path"
+import { getConfig } from "@lingui/conf"
+import { createCompiledCatalog, getCatalogs, getCatalogForFile } from "@lingui/cli/api"
+
+type LinguiConfigOpts = {
+  cwd?: string;
+  configPath?: string;
+  skipValidation?: boolean;
+}
+type SnowpackLoadOpts = {
+  filePath: string
+}
+function extractLinguiMessages(_?, linguiConfig: LinguiConfigOpts = {}) {
+  const strict = process.env.NODE_ENV !== 'production'
+  const config = getConfig(linguiConfig)
```

Comment:
For me has sense outside load because if config is wrong Will fail with all files. If we use load we get the config on each file (slower and could be desynchronization

Reply:
OK, I was thinking the same 👌